app/assets/stylesheets/base/typography.scss

```diff
@@ -33,7 +33,7 @@
     @apply mb-2;
 
     &:last-of-type {
-      @apply mb-0;
+      @apply mb-2;
```

Comment:
This worked but there may be a better fix.

I tried adding a tailwind margin setting to the `base_classes` of app/components/button_component.rb and to the agreement text
```
    <p class="text-sm italic"><%= t('space.agreements.help_text') %></p>
```
in app/views/spaces/edit.html.erb but it did not fix the issue.